Content.Shared/_RMC14/Clothing/RMCClothingSystem.cs

```diff
@@ -98,6 +104,19 @@ private void OnRequireEquippedBeingEquippedAttempt(Entity<ClothingRequireEquippe
         args.Reason = ent.Comp.DenyReason;
     }
 
+    private void OnDropped(Entity<ClothingComponent> ent, ref DroppedEvent args)
+    {
+        var slots = _inventory.GetSlotEnumerator(args.User);
+        while (slots.MoveNext(out var slot))
+        {
+            if (slot.ContainedEntity is not { } contained)
+                continue;
+
+            if (TryComp<ClothingRequireEquippedComponent>(contained, out var requiresEquipped) && requiresEquipped.AutoUnequip && _whitelist.IsValid(requiresEquipped.Whitelist, ent.Owner))
```

Comment:
I don't think this matters for current usages of this but you would need to check if no other item fulfills the whitelist before force unequipping